app/controllers/concerns/json_renderer.rb

```diff
@@ -40,7 +40,7 @@ def add_includes(root, resources, includes)
 
   private
 
-  def render_as_json(namespace, resource, status: :ok, allowed_includes: [])
+  def render_as_json(namespace, resource, status: :ok, allowed_includes: [], pagy: nil)
```

Comment:
make pagy a required argument so all call-sites have to consider adding `@pagy` or `nil`

Reply:
done